docs/mmteb/points.md

```diff
@@ -2,24 +2,24 @@
 
```

Comment:
The points system changed, now you just need to add a json file named as your PR number. Here is [an example](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points/429.jsonl) and the [docs](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md).

Reply:
Done!